packages/workflow/src/workflow.ts

```diff
@@ -112,11 +113,14 @@ function timerNextHandler(input: TimerInput) {
     }
     activator.pushCommand({
       startTimer: {
-        seq: input.seq,
-        startToFireTimeout: msToTs(input.durationMs),
+        seq,
+        startToFireTimeout: msToTs(durationMs),
+      },
+      userMetadata: options && {
+        summary: options.summary ? activator.payloadConverter.toPayload(options.summary) : undefined,
```

Comment:
What if `summary` is an empty string? As it is now, it will be replaced by `undefined`. Not sure what would be the desired result.

Reply:
And we don't care about empty strings, then should instead move the terniary out of this object:

```
      userMetadata: options?.summary
        ? {
            summary: activator.payloadConverter.toPayload(options.summary),
          }
        : undefined,
```

Reply:
I think having an empty `summary` string replaced with `undefined` is reasonable. I don't see a reason why having an empty summary would be useful (or any more useful than undefined).